spec/metadata_json_deps_spec.rb

```diff
@@ -69,7 +69,7 @@
       let(:module_name) { 'theforeman/motd' }
       let(:module_version) { '< 0.1.0' }
 
-      it { expect { subject }.to output(%r{\AChecking .+puppet-module.+json\n  theforeman/motd \(< 0\.1\.0\) doesn't match 1\.0\.0\Z}).to_stdout }
+      it { expect { subject }.to output(%r{\AChecking .+puppet-module.+json\n  theforeman/motd \(< 0\.1\.0\) doesn't match 2\.0\.0\Z}).to_stdout }
```

Comment:
I think it's not ideal that we test against actual data, but adjusting the version here was the easiest/quickest fix.